spec/services/coronavirus/pages/draft_updater_spec.rb

```diff
@@ -54,4 +54,25 @@
       )
     end
   end
+
+  describe "#discard" do
+    it "raises an error if there isn't a draft to discard" do
+      stub_request(:post, "#{Plek.current.find('publishing-api')}/v2/content/#{page.content_id}/discard-draft")
+        .and_raise(GdsApi::HTTPUnprocessableEntity)
```

Comment:
Like a previous comment, it feels more idiomatic for request stubs to return status' that are converted by the underlying library to exceptions. You could also simplify this to:
```suggestion
      stub_any_publishing_api_discard_draft.to_return(status: 422)
```

Reply:
This is much nicer! Thanks.